src/CycloneDX.Spdx/Models/v2_3/ChecksumAlgorithm.cs

```diff
@@ -0,0 +1,60 @@
+// This file is part of CycloneDX Library for .NET
+//
+// Licensed under the Apache License, Version 2.0 (the “License”);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an “AS IS” BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX_License_Identifier: Apache_2.0
+// Copyright (c) OWASP Foundation. All Rights Reserved.
+
+using System;
+using System.Xml.Serialization;
+
+namespace CycloneDX.Spdx.Models.v2_3
+{
+    public enum ChecksumAlgorithm
```

Comment:
it's the same case as for PACKAGE_MANAGER, technically according to the [documentation](https://spdx.github.io/spdx-spec/v2.3/package-information/#710-package-checksum-field), instead of underscores there should be hyphen, but I wasn't able to find a solution for this issue

Reply:
Again I tried applying the changes in #268 but now for the ChecksumAlgorithm that contain a hyphen and this also worked here 😌
I can commit the updated version if you want @andreas-hilti